.jenkins/pytorch/test.sh

```diff
@@ -10,6 +10,14 @@ COMPACT_JOB_NAME="${BUILD_ENVIRONMENT}"
 # Get fully qualified path using realpath
 CUSTOM_TEST_ARTIFACT_BUILD_DIR=$(realpath "${CUSTOM_TEST_ARTIFACT_BUILD_DIR:-${PWD}/../}")
 
+TORCH_INSTALL_DIR=$(python -c "import site; print(site.getsitepackages()[0])")/torch
+TORCH_LIB_DIR="$TORCH_INSTALL_DIR"/lib
+TORCH_TEST_DIR="$TORCH_INSTALL_DIR"/test
```

Comment:
i am not familiar with wheel packaging but is there a reason why after wheel packaging those test files are in `/test` folder where as in regular build they are in `/bin`?

Reply:
This would go back to build stage of how the wheel package is generated. The organization of the build/ folder and wheel is a bit different (for example atest, basic ... are in `build/bin` but then are copied to `envs/pytorch/lib/python3.8/site-packages/torch/test/` Maybe at some point we should refactor a bit the wheel building step